guidelines/terms/20/relative-luminance.html

```diff
@@ -40,6 +40,8 @@
    </p>
 
   </div>
+
+  <p class="note">Before May 2021 the value of 0.04045 in the definition was different. It was taken from an older version of the specification and has been updated. It should have no practical effect on the calculations.</p>
```

Comment:
worth mentioning the old value, to provide a historical record (and to have a reference for any tools that may have used this value internally)

`was different (0.03928).`